include/libdnf/transaction/transaction_item_action.hpp

```diff
@@ -28,19 +28,21 @@ namespace libdnf::transaction {
 // Any time you add a new action, change functions that resolve reasons,
 // because removed items (RPMs) must be excluded from reason resolution:
 // * Package.cpp - Package::resolveTransactionItemReason
+// TODO(jmracek) Copy enum to SWDB including their string definitions and already removed values to keep compatibility
+// Then transform old database data to transform stored reverse action to poplaced action and remove all unused action from code
```

Comment:
Typo: poplaced -> REPLACED (caps make it clearer)

Reply:
Fixed